L1Trigger/TrackTrigger/src/TTStubAlgorithm_cbc3.cc

```diff
@@ -15,9 +15,10 @@ void TTStubAlgorithm_cbc3<Ref_Phase2TrackerDigi_>::PatternHitCorrelation(
     bool &aConfirmation,
     int &aDisplacement,
     int &anOffset,
-    float &anROffset,
     float &anHardBend,
-    const TTStub<Ref_Phase2TrackerDigi_> &aTTStub) const {
+    const TTStub<Ref_Phase2TrackerDigi_> &aTTStub) const
+// Removed real offset.  Ivan Reid 10/2019
```

Comment:
please inform developers that a commit message is a more appropriate place for this kind of remark, rather than a dated comment in the code

Reply:
ok.